src/content/docs/guides/contribution/changelog.mdx

```diff
@@ -2,31 +2,115 @@
-We maintain automatically generated changelog with *user-facing* change
-descriptions.
-
-The procedure for adding a changelog entry looks as follows.
-
-1. Open your pull request with your proposed changes
-2. Go to the [`changelog`](https://github.com/tenzir/tenzir/tree/main/changelog)
-   directory in the top-level repository directory and navigate to the
-   `next` sub-directory.
-3. Choose a category for your changes and go to the corresponding sub-directory:
-   - **Feature** → `features`
-   - **Bug Fix** → `bug-fixes`
-   - **Change** → `changes`
-   - **Breaking Change** → `breaking-changes`
-4. Add a file with the following filename structure: `X1[-X2-...-Xn][--Y].md`.
-   where `X` is either a PR number in the tenzir/tenzir repository or a GitHub
-   account name. We only include account names for external contributions
-   outside from the Tenzir core team. Everything after the two dashes `--` is an
-   optional description to clarify what the change was about.
-
-Every installation and build of Tenzir contains the amalgamated CHANGELOG.md
-file. To view that directly, open `path/to/build/CHANGELOG.md` for your Tenzir
-build.
+We maintain automatically generated [changelog](../../../changelog) with
+*user-facing* change descriptions.
+
+The source for this generated page is a set of Markdown files in the
+[`/changelog`](https://github.com/tenzir/docs/tree/main/changelog) directory of
+our docs repo. There exists a directory for `node` and `platform` with the
+following structure:
+
+import { FileTree } from '@astrojs/starlight/components';
+
+<FileTree>
+- changelog
+  - node
+    - next
+      - 2uwQMJ50ob7ijy9NS9XdWh5yfsc.md
+      - 2uwQnMTpRSDe8bB0mJXFkyujped.md
+    - v5.0.0
+      - some-feature.md
+  - platform
+    - next
+      - 2uwac8f6MEARnzJ9mORKqtm9pwA.md
+</FileTree>
+
+Each product (`node`, `platform`) has subdirectories for each version (e.g.,
+`v5.0.0`), with `next` referring to the currently unreleased version
+accumulating changes on the main branch. Within each version directory exists a
+set of `.md` files representing individual changelog entries.
+
+## Write a changelog entry
+
+To add an entry to the changelog, follow these steps:
+
+1. **Navigate to the correct directory:** Go into the `changelog/` directory,
+   then the appropriate product subdirectory (`node` or `platform`), and finally
+   the version subdirectory corresponding to the release your change will be
+   part of. For changes currently on the `main` branch that haven't been
+   released yet, use the `next` directory.
+
+   - Example: A feature for the upcoming Node release goes into
+      `changelog/node/next/`.
+   - Example: A bug fix for Platform v1.2.0 would go into
+      `changelog/platform/v1.2.0/`.
+
+2. **Create a new Markdown file:** Create a new file ending in `.md`. While the
+   exact filename doesn't impact the final output (our examples use IDs), using
+   a descriptive name can be helpful (e.g., `add-frobnication-widget.md`).
```

Comment:
I'm currently using KSUIDs here as filenames. We could also use UUIDv7, which has the same properties but is slightly more verbose, yet may have better tooling support.

@dominiklohmann any preference? We also need to discuss the workflow in the respective repos that clone the changelog.